src/Output/ProgressBar.php

```diff
@@ -298,7 +323,38 @@ protected function percentageFormatted(float $percentage): string
      */
     protected function labelFormatted(string $label): string
     {
-        return "\n" . $this->cr() . $this->cursor->eraseLine() . $label;
+        return "\n" . $label;
+    }
+
+    /**
+     * Format the output of the progress bar by placing the label in the right place (top, right, bottom or left)
+     */
+    protected function progressBarFormatted(string $bar, string $number, string $label): string
+    {
+        if ($this->options['showPercentage'] !== true) {
```

Comment:
`if (!$this->options['showPercentage'])`

Reply:
Done